Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixInfo.tsx

```diff
@@ -0,0 +1,121 @@
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import { Button, Modal, ModalBody, ModalFooter, ModalHeader, Table } from "reactstrap";
+
+interface IProps {
+    bigfixId: string;
+}
+
+interface IState {
+    bigfixModal: boolean;
+    computerInfo: object;
+    isFetched: boolean;
+}
+
+export default class EquipmentDetails extends React.Component<IProps, IState> {
+    public static contextTypes = {
+        fetch: PropTypes.func,
+        permissions: PropTypes.array,
+        router: PropTypes.object,
+        team: PropTypes.object
+    };
+
+    public state = { bigfixModal: false, computerInfo: {}, isFetched: false };
+
+    public modalToggle = () => {
+        this.setState({
+            bigfixModal: !this.state.bigfixModal
+        });
+    };
+
+    public render() {
+        return (
+            <>
+                <label>
+                    Bigfix Id
+                    <a
+                        onClick={() => {
+                            this.modalToggle();
+                            this.getBigFixComputerInfo(this.props.bigfixId || "");
+                        }}
+                    >
+                        <i className=" ml-2 fas fa-info-circle" />
+                    </a>
+                </label>
+                {this.renderBigFixModal()}
+            </>
+        );
+    }
+
+    private renderBigFixModal = () => {
```

Comment:
not a huge deal, but for consistency i'd name your private methods starting with an underscore, e.g. `_renderBigFixModal()`

Reply:
fixed